cocos/storage/local-storage/LocalStorage.h

```diff
@@ -46,7 +46,7 @@ void CC_DLL localStorageFree();
 void CC_DLL localStorageSetItem( const std::string& key, const std::string& value);
 
 /** Gets an item from the JS. */
-std::string CC_DLL localStorageGetItem( const std::string& key );
+bool CC_DLL localStorageGetItem( const std::string& key, std::string *outItem );
```

Comment:
Why modify like this?
It will break compatibility.
Why not add a new function if it is needed.